src/main/java/ua/su/service/impl/FindSomethingInClinic1.java

```diff
@@ -0,0 +1,25 @@
+package ua.su.service.impl;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ua.su.repository.ClinicRepository;
+import ua.su.repository.MedicalProcedureRepository;
+import ua.su.service.FindSomethingByCriteria;
+
+import java.util.List;
+
+@Service
+public class FindSomethingInClinic1 extends ClinicServiceImpl implements FindSomethingByCriteria {
```

Comment:
It is better to rename it to "ClinicServiceSearchSql" it is going to be more detail described

Reply:
Thank you! I renamed the class 'FindSomethingInClinic1.java' to 'ClinicServiceSearchSql'.